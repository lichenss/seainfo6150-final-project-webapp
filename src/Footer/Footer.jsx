import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Footer.module.css'
const Footer = (props) => {
    return (
            <footer>
                 <div className={styles.links}>
                 <Link className={styles.link} to="/contact-us">
                     <h3 >Contact Us</h3>
                     </Link>
                     <Link className={styles.link} to="/about-us">
                         <h3 >About Us</h3>
                    </Link>
                    </div>
            </footer>
    );
};

export default Footer;