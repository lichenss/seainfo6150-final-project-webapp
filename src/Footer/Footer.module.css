
.footer{
    background-color: #f7f3f3;
    left: auto;
    right: 0;
    position: bottom;
    width: 100%;
    z-index: 1;
}
.links{
    flex:1;
    display: flex;
    justify-content: center;
    
}
.link{
    margin-left: 50px;
    margin-right: 50px;
    color:#17488D;
}