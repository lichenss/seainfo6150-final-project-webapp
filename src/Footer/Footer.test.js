import React from "react";
import Footer from "./Footer";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
describe("Footer tests", () => {
    it("renders correctly", () => {
      const { container } = render(
      <BrowserRouter>
      <Footer />
      </BrowserRouter>);
      expect(container).toMatchSnapshot();
    });
  });