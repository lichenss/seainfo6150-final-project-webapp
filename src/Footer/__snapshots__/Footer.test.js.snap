// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests renders correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="links"
    >
      <a
        class="link"
        href="/contact-us"
      >
        <h3>
          Contact Us
        </h3>
      </a>
      <a
        class="link"
        href="/about-us"
      >
        <h3>
          About Us
        </h3>
      </a>
    </div>
  </footer>
</div>
`;
